product-service/service/products-service.js

```diff
@@ -0,0 +1,10 @@
+import { PRODUCTS } from '../data/products';
+
+const findById = (id) => PRODUCTS.find((product) => product.id === id);
```

Comment:
I suggest to return Promise if you want to use `ProductsService` with `await` in lambdas